go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/red-hat-storage/ocs-client-operator
 
-go 1.22.0
-
-toolchain go1.22.3
+go 1.22.5
```

Comment:
Can you please check we have this version in downstream if you are planning for backport to 4.17

Reply:
I'm not thinking of backport, nevertheless d/s does support this version.